clients/python/pyproject.toml

```diff
@@ -44,7 +44,7 @@ sphinx-autobuild = ">=2021.3.14,<2025.0.0"
 pytest = ">=7.4.2,<9.0.0"
 coverage = { extras = ["toml"], version = "^7.3.2" }
 pytest-cov = ">=4.1,<6.0"
-ruff = "0.5.2"
+ruff = "^0.5.2"
```

Comment:
I believe Dependabot will try to <0.6.0 at the next round of updates, but either works for me